packages/task/src/node/task.ts

```diff
@@ -43,6 +44,9 @@ export abstract class Task {
     /** Terminates the task. */
     abstract kill(): Promise<void>;
 
+    /** Initializes the client connection for the task. */
+    abstract initClientConnection(connection: MessageConnection): void;
```

Comment:
Should the required events to hook be described in the jsdoc?

We are just handed a `MessageConnection` object, but no clue as to what send...

Reply:
I'm all for documentation.  I was following the convention that seems to be used here of no comments and just letting people look at the default implementations.  I will add some jsdoc here.

The actual required events will depend on what the developer is doing that requires the developer to override the default implementation from ProcessTask.